app/templates/my_questions.html

```diff
@@ -16,6 +16,8 @@ <h4>My Questions</h4>
 					<li><a class="dropdown-item" href="{{url_for('instructor.create_new_question', question_type='multiple-selection')}}">Multiple Selection</a></li>
 					<li><a class="dropdown-item" href="{{url_for('instructor.create_new_question', question_type='code-jumble')}}">Code Jumble</a></li>
 					<li><a class="dropdown-item" href="{{url_for('instructor.create_new_question', question_type='single-line-code')}}">Single Line Code</a></li>
+					<li><a class="dropdown-item" href="{{url_for('instructor.create_new_question', question_type='fill-in-the-blank')}}">Fill In The Blank (Auto-Graded)</a></li>
```

Comment:
Let's use the extract phrase "Fill In The Blank(s)" to give a quick indication that there may be multiple blanks to fill in. There isn't any need to put the "Auto Graded" as the expectation is that FITB questions should be auto-graded.